docs/src/utility_calculators.md

```diff
@@ -0,0 +1,97 @@
+# Utility calculators
+
+We supply special calculators that can be used to alter input 
+
+All of these calculators are defined in submodule [UtilityCalculators](@ref). To use them just add
+
+```julia
+using AtomsCalculators.UtilityCalculators
+```
+
+## Combination Calculator
+
+[CombinationCalculator](@ref) can be used to combine separate calculators to a single calculator. This is usful when results of several calculators are added together.
+
+Here are couple of ways to create CombinationCalculator
+
+```julia
+CombinationCalculator(calc1, calc2)
+CombinationCalculator(calc1, calc2, calc3...)
+CombinationCalculator(calc1, calc2; multithreading=true)
+```
+
+By default multithreading is off, but giving kword `multithreading=true` toggles it on. This means that all calculators are launched at the same time.
+
+[CombinationCalculator](@ref) has an additional feature that allows it to edit what keywords are passed down on the calculators. This allows e.g. generating data that can be used by all the calculators.
+
+The default is passing keywords as they are. There are two ways to change this, fist is global setting with [generate_keywords](@ref) function or by givin keyword `keywordgenerator`. Global option is intended e.g. for package authors to make global settings. You could e.g. use it to calculate pair list for pair potentials:
+
+```julia
+function AtomsCalculators.UntilityCalculators.generate_keywords(
+    sys, 
+    pp::PairPotential...; 
+    kwargs...
+)
+    cutoff = cutoff_radius(pp[begin])
+    if all( c ->  cutoff_radius(c) ≈ cutoff, pp )
+        plist = PairList(sys, cutoff)
+        return (; :plist => plist, kwargs...)
+    else
+        return kwargs
+    end
+end
```

Comment:
Can't this be solved, by the convention that all kwargs are passed to all calculators in a CombinationCalculator?

Reply:
That is the default behavior. This was just an example on how to customize it - it is in the docs after all.